### DISCUSSION TITLE
Suggestion: Require enchantment to function

### DISCUSSION THREAD

**Discussion by kepler-gray** [RakambdaOrg/FallingTree#695] [Ideas]
I have a suggestion for an idea I'd like to see that I've never seen in a treecapitator-esque plugin before: Require the axe to have a specific custom enchantment ("Treefeller" or something? idk I'm not creative) to function. I do love the idea of this plugin but an option to gate its functionality behind an enchantment would make it feel much less cheaty to me, in a similar manner to the VeinMiner mod.

**Comment by Rakambda** [comment]
This is already possible in the settings. There's a whole enchantment section.
Gotta be careful though, it'll require the mod to be on both server AND client (instead of just server).

**Comment by kepler-gray** [comment]
Wow, I can't believe I missed that, sorry about that. Thanks! Just looking for singleplayer so that's not an issue for me :) have a good one!
